lib/ansible/modules/cloud/misc/qemu_img.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2013, Jeroen Hoekx <jeroen.hoekx@dsquare.be>, Toshaan Bharvani <toshaan@vantosh.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = { 'status': ['preview'],
+                     'supported_by': 'community',
+                     'version': '1.0'}
+
+DOCUMENTATION = '''
+---
+author:
+  - Jeroen Hoekx (@jhoekx)
+  - Toshaan Bharvani (@toshywoshy)
+module: qemu_img
+short_description: Create qemu images
+description:
+  - "This module creates images for qemu."
+version_added: "2.3"
+options:
+  dest:
+    description:
+    - The image file to create or remove
+    required: true
+  format:
+    description:
+    - The image format - default qcow2
+    default: qcow2
+  size:
+    description:
+    - The size of the image in megabytes.
+  options:
+    description:
+    - Comma separated list of format specific options in a name=value format.
+  size:
+    description:
+    - The size of the image
+  grow:
+    choices: [ "yes", "no" ]
+    defaults: "yes"
+    description:
+    - Whether the image should grow
+  shrink:
+    choices: [ "yes", "no" ]
+    defaults: "no"
+    description:
+    - Whether the image should shrink
+  state:
+    choices: [ "absent", "present"]
+    description:
+    - If the image should be present or absent - default present
+  notes:
+    - This module does not change the type/format of the image
+    - This module does not take snapshots, and should be implemented seperate
+'''
+
+EXAMPLES = '''
+examples:
+  - description: Create a raw image of 5M.
+    code: qemu_img dest=/tmp/testimg size=5M format=raw
+
+  - description: Enlarge the image to 6G.
+    code: qemu_img dest=/tmp/testimg size=6G format=qcow2
+
+  - description: Shrink the image by 3G
+    code: qemu_img dest=/tmp/testing size=-3G format=qcow2
+
+  - description: Remove the image
+    code: qemu_img dest=/tmp/testimg state=absent
```

Comment:
The way to do examples changed. Now your EXAMPLES section can simply be a list of tasks, or a play.